src/main/java/ru/sevsu/rc_manager/sound/processor/SoundReceiver.java

```diff
@@ -0,0 +1,66 @@
+package ru.sevsu.rc_manager.sound.processor;
+
+import jakarta.annotation.PostConstruct;
+import lombok.RequiredArgsConstructor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import ru.sevsu.rc_manager.sound.handler.SoundHandler;
+
+import javax.sound.sampled.*;
+import java.io.ByteArrayOutputStream;
+
+@Service
+@RequiredArgsConstructor
+public class SoundReceiver {
+    private final SoundProcessor soundProcessor;
+    private final SoundHandler soundHandler;
+    private final SoundFormat soundFormat;
+    private static final Logger log = LoggerFactory.getLogger(SoundReceiver.class);
+
+    @Value("${sound.min-duration}")
+    private int minDuration;
+
+    @PostConstruct
+    public void processAudio() {
```

Comment:
Your class is called SoundReceiver. It would be logical to have method receiveSound. Do not use synonyms.